Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
+gem 'fastimage'
```

Comment:
I don't think it's needed as img_checker would get it for you. However I am not sure.

Reply:
It doesn't work without it, I tried

Reply:
It will work, I double checked.

Reply:
yes it does work

Reply:
Then remove it please